### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by antgonza**
Improving the README page and also adding qiita-philosophy to the tutorials.

### CODE REVIEW

**Comment by josenavas** [comment]
@antgonza the docs are not building correctly here - not sure what is the error, but one guess is that the folder "qiita-philosophy" needs to be added to setup.py

File: qiita_pet/support_files/doc/source/qiita-philosophy/index.rst [hunk 1]
```diff
@@ -0,0 +1,82 @@
+Qiita Philosophy
+================
+
+Plugin ideology
+---------------
+
+Qiita can be seen as an analytical pipeline broker that can apply any specific
+pipeline, tool, or script to any of its stored data. All the analytical
+pipelines are autonomous and independently developed, and tested, which
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/qiita-philosophy/index.rst]: `and independently` -> `independently`

File: qiita_pet/support_files/doc/source/qiita-philosophy/index.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+The current plugins available are:
+
+* Target gene processing via `QIIME <http://qiime.org>`__: sequence
+  demultiplexing, and QC of SFF, fasta/qual, fastq and per sample fastq.
+  Closed reference OTU picking.
+
+A Study
+-------
+
+Qiita’s main entity is the idea of Study. A study can have many samples, with
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/qiita-philosophy/index.rst]: of Study -> of a study

**Comment by ElDeveloper** [comment]
:+1:

**Pull request merged**